### DISCUSSION TITLE
Missing command ?

### DISCUSSION THREAD

**Discussion by mybouhssina** [Z3d0X/filament-fabricator#123] [Q&A] [answered]
Hi, it could be me, but I couldn't get the plugin to work without manually running:
php artisan filament:assets
Before that I had the filament-peek.css missing
is it missing from the documentation or I missed something ?

**Comment by Z3d0X** [comment, marked as answer]
Hi, this command will automatically be run, if you have `php artisan filament:upgrade` command hooked on composer post-autoload, like the docs suggest.

```php
//composer.json
"post-autoload-dump": [
    // ...
    "@php artisan filament:upgrade"
],
```

https://filamentphp.com/docs/3.x/panels/installation#upgrading
